plugin.xml

```diff
@@ -47,7 +47,7 @@
             <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE"/>
         </config-file>
 
-        <framework src="com.segment.analytics.android:analytics:4.2.6"/>
+        <framework src="com.segment.analytics.android:analytics:4.3.0-RC1"/>
```

Comment:
okay, if we are not going to use `RC's` there is no new analytics version for android in b/w.
https://github.com/segmentio/analytics-android